web-admin/src/routes/[organization]/[project]/-/deployment/+page.svelte

```diff
@@ -13,7 +16,7 @@
 <div class="flex flex-col items-center">
   <div class="flex space-x-10 border-b border-gray-200 w-full px-12 py-5">
```

Comment:
Would suggest you use grid here, not flex, and explicitly set these column widths according to the mocks. It'll be much more usable and easier to tweak

Reply:
Good call. I'll change the layout in a different PR, but just added a max-width & truncate here, so there's no excessive overflow.